src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -7934,36 +7945,33 @@ private BoundExpression BindIndexedPropertyAccess(SyntaxNode syntax, BoundExpres
 
                 ImmutableArray<PropertySymbol> candidates = propertyGroup.ToImmutable();
 
-                if (!analyzedArguments.HasErrors)
+                if (TryBindIndexOrRangeImplicitIndexer(
+                    syntax,
```

Comment:
Nit: these being as intended as the method itself is messing with me a bit. Consider indenting.